src/trino/truncate_impl.rs

```diff
@@ -172,7 +215,7 @@ pub(super) struct truncate_doubleFunc {
 }
 
 impl truncate_doubleFunc {
-    pub fn new() -> Self {        
+    pub fn new() -> Self {
```

Comment:
Ugh... let's fix the generator script so that it calls `cargo fmt`? Or else we'll just have non-stop garbage like this in every PR...

Also we should set up the CI properly

Reply:
Yes. Let me fix it.